packages/cashier/src/pages/on-ramp/on-ramp-providers.ts

```diff
@@ -41,7 +41,10 @@ const createBanxaProvider = (store: OnRampStore) => ({
                     const { url } = response.service_token.banxa;
 
                     if (url) {
-                        window.open(url);
```

Comment:
since window.open is not supported on safari, did this hacky way to bypass Safari blocking any window. open() function during an async call to which provides "better" security